internal/operators/osc/operator.go

```diff
@@ -0,0 +1,178 @@
+package osc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/featuresupport"
+	"github.com/openshift/assisted-service/internal/operators/api"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
+	logutil "github.com/openshift/assisted-service/pkg/log"
+	"github.com/sirupsen/logrus"
+)
+
+type operator struct {
+	log    logrus.FieldLogger
+	config *Config
+}
+
+var Operator = models.MonitoredOperator{
+	Name:             Name,
+	Namespace:        Namespace,
+	OperatorType:     models.OperatorTypeOlm,
+	SubscriptionName: Subscription,
+	TimeoutSeconds:   60 * 60,
+}
+
+func NewOscOperator(log logrus.FieldLogger) *operator {
+	cfg := Config{}
+	err := envconfig.Process(common.EnvConfigPrefix, &cfg)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	return &operator{
+		log:    log,
+		config: &cfg,
+	}
+}
+
+// GetName reports the name of an operator this Operator manages
+func (o *operator) GetName() string {
+	return Operator.Name
+}
+
+func (o *operator) GetFullName() string {
+	return FullName
+}
+
+func (o *operator) GetDependencies(cluster *common.Cluster) ([]string, error) {
+	return []string{Operator.Name}, nil
```

Comment:
Should be 

return make([]string, 0), nil

Reply:
Updated